build.gradle

```diff
@@ -60,3 +60,36 @@ allprojects {
 task clean(type: Delete) {
     delete rootProject.buildDir
 }
+
+apply plugin: 'maven-publish'
+
+def getArtificatId = { ->
+    return "rxandroidble"
+}
+
+def getArtificatVersion = { ->
+    return "1.1"
+}
+
+publishing {
+    publications {
+        app(MavenPublication) {
+            groupId 'com.github.rxandroidble'
+            artifactId getArtificatId()
+            version getArtificatVersion()
+
+            artifact('./rxandroidble/build/outputs/aar/rxandroidble-debug.aar') {}
```

Comment:
why did you choose `debug` here and not `release` ?

Reply:
When I built this locally, I only got the debug file. Never even saw a release file.